iOS/MMIWG2S/ViewControllers/FacesViewController.swift

```diff
@@ -1,357 +0,0 @@
-/*
- * Copyright 2019 Google LLC. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import UIKit
```

Comment:
fwiw, commenting out the code should have the same size impact

Reply:
Hmm, I felt comfortable removing it entirely, because the phones this was being used on are 5 or more years old. I guess the new iPhone SE doesn't have Face ID, now that I check. Thanks for making me think that over :)